src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/OrchardFieldNameConverter.cs

```diff
@@ -1,28 +1,21 @@
 using System;
 using GraphQL.Conversion;
+using GraphQL.Types;
 
 namespace OrchardCore.Apis.GraphQL
 {
-    public class OrchardFieldNameConverter : IFieldNameConverter
+    public class OrchardFieldNameConverter : INameConverter
     {
-        private readonly IFieldNameConverter _defaultConverter = new CamelCaseFieldNameConverter();
+        private readonly INameConverter _defaultConverter = new CamelCaseNameConverter();
 
-        public string NameFor(string field, Type parentType)
+        public string NameForArgument(string fieldName, IComplexGraphType parentGraphType, FieldType field)
         {
-            var attributes = parentType?.GetCustomAttributes(typeof(GraphQLFieldNameAttribute), true);
```

Comment:
i cant remember what  atm.. but surely GraphQLFieldNameAttribute does something specific; otherwise do we even need a custom name converter @Jetski5822 - can you remember ?

Reply:
fixed in ##10782